Yoast/wpseo-news
# Fix broken link in news sitemap

Fix broken link in news sitemap and make text better readable for screen readers.

## Summary


This PR can be summarized in the following changelog entry:
Fixed broken link in news sitemap

## Relevant technical choices:

* As the current link text was `here`, I changed the wording altogether, to make sure the link is more accessible for screen readers.

## Test instructions

This PR can be tested by following these steps:

* Create a new article
* Go to the News sitemap
* Check if the link for more information about XML News sitemaps works.

Fixes #168
